src/language-css/printer-postcss.js

```diff
@@ -120,6 +120,7 @@ function genericPrint(path, options, print) {
       return isInlineComment ? text.trimEnd() : text;
     }
     case "css-rule": {
+      const nodeSeparator = singleLineRule(node) ? " " : hardline;
```

Comment:
```suggestion
      const nodeSeparator = singleLineRule(node) ? line : hardline;
```

This should (?) make it still break when the rule itself breaks.